src/Drupal/Driver/DrushDriver.php

```diff
@@ -189,11 +189,27 @@ public function clearStaticCaches() {
   }
 
   /**
+   * Decodes JSON output returned by Drush.
+   *
+   * It will clean up any junk that may have appeared before the JSON.
+   *
+   * @param string $output
+   *   The output from Drush.
+   * @return object
+   *   The decoded JSON object.
+   */
+  protected function decodeJsonOutput($output) {
+    // Remove anything before the first '{'.
+    $output = preg_replace('/^[^\{]*/', '', $output);
```

Comment:
You are right that we do not need the 's' modifier here.

`[^\}]` will match anything except `}`, so it will not match after the first `}`.  The `$` ensures that the matching pattern must be at the end of the buffer.